src/python/turicreate/__init__.py

```diff
@@ -90,7 +90,7 @@
 # we can use top-level lazy import for them
 distances = _LazyModuleLoader('turicreate.toolkits.distances')
 nearest_neighbors = _LazyModuleLoader('turicreate.toolkits.nearest_neighbors')
-topci_model = _LazyModuleLoader('turicreate.toolkits.topci_model')
+topic_model = _LazyModuleLoader('turicreate.toolkits.topic_model')
```

Comment:
Hmmm, I'm surprised no test is for this module and I even passed the smoke test. Probably this module is no longer in use?

Reply:
Probably still leave it in for people to use if they want.